src/System.Reflection.Metadata.Cil/src/Decoder/CilTypeProvider.cs

```diff
@@ -229,9 +237,10 @@ public CilType GetTypeFromReference(TypeReferenceHandle handle, bool? isValueTyp
             return type;
         }
 
+        [CLSCompliant(false)]
         public string GetParameterList(MethodSignature<CilType> signature, ParameterHandleCollection? parameters = null)
         {
-            ImmutableArray<CilType> types = signature.ParameterTypes;
+            var types = signature.ParameterTypes;
```

Comment:
See rule 10 in our coding guidelines https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md